index.ts

```diff
@@ -88,7 +88,7 @@ async function run() {
       repo: repo,
       issue_number: number
     })
-  ).data.find(comment => comment.body.includes(footer));
+  ).data.find(comment => comment.body.includes(header) && comment.user.login === "github-actions[bot]");
```

Comment:
Will the bot always be the author? Or do we get the name from the `gh-token` option?

Basically, if we set `gh-token` to be some other token, the name will change and this line will break.

Reply:
`github-actions[bot]` is tied to the `GITHUB_TOKEN` if we use a different token that is associated with a different account like our `ACTIONS_PAT` then it is that author who makes the comment

![ifixit systems comment](https://user-images.githubusercontent.com/22064420/199291550-9ee05c8c-0839-42e5-bda9-2d861cdbc924.png)

Reply:
> Basically, if we set gh-token to be some other token, the name will change and this line will break.

If we need to, I think we can add a new `comment-author` input in the action inputs and pass that from the workflow. But I don't think it's necessary since we'll be mostly using `${{ secrets.GITHUB_TOKEN }}`?

Reply:
The reason we would need to use a different `token` is if we need to do more then commenting like committing.

Reply:
Sounds good. I'm good with it.